pkg/drivers/generic/generic.go

```diff
@@ -20,33 +20,32 @@ const (
 )
 
 var (
-	columns = "kv.id as theid, kv.name, kv.created, kv.deleted, kv.create_revision, kv.prev_revision, kv.lease, kv.value, kv.old_value"
+	columns = "kv.id AS theid, kv.name, kv.created, kv.deleted, kv.create_revision, kv.prev_revision, kv.lease, kv.value, kv.old_value"
 	revSQL  = `
-		SELECT rkv.id
-		FROM kine rkv
-		ORDER BY rkv.id
-		DESC LIMIT 1`
+		SELECT MAX(rkv.id) AS id
+		FROM kine AS rkv`
```

Comment:
Why are we changing this? The result looks to be the same, logically, but is not the ORDER BY more efficient?

Reply:
Hmm, according to some rudimentary searching the performance should be equivalent for non-null, indexed columns

Reply:
The behavior appears to be engine-specific, but in looking at the query plan it did eliminate a reported table scan.